ext/hal/st/stm32cube/stm32l1xx/soc/stm32l152xe.h

```diff
@@ -91,6 +91,7 @@ typedef enum
   BusFault_IRQn               = -11,    /*!< 5 Cortex-M3 Bus Fault Interrupt                         */
   UsageFault_IRQn             = -10,    /*!< 6 Cortex-M3 Usage Fault Interrupt                       */
   SVC_IRQn                    = -5,     /*!< 11 Cortex-M3 SV Call Interrupt                          */
+  SVCall_IRQn                 = -5,     /*!< 11 Cortex-M3 SV Call Interrupt                          */

```

Comment:
We're not supposed to modify stm32cube packages, unless we identify a genuine, non zephyr specific, bug.
Reason is we don't want to maintain zephyr specific update in these packages and whatever modification we do should be eventually integrated in stm32cube packages.

In this particular case the line is thin, but it seems  SVC_IRQn is being used as enum value for SV Call Interrupt for all C-M0/C-M0+ SoCs (F0/L0), while L1 is M3 based.

Note: This issue was not faced on these SoCs because the breaking code is under CPU_CORTEX_M_HAS_BASEPRI which is not available for M0 based SoCs.

So I agree to consider this as a Cube package issue, but please extend it to all impact STM32L1 SoCs and submit it as a separate PR. Also, you'll need to additionally update README file for this package (ext/hal/st/stm32cube/stm32l1xx/README).

Reply:
Please use ST Bug tracker ID: 66029 when updating the README

Reply:
Please review https://github.com/zephyrproject-rtos/zephyr/pull/15744